src/util.cpp

```diff
@@ -78,6 +78,14 @@
 #include <openssl/conf.h>
 #include <thread>
 
+#ifdef WIN32
```

Comment:
As this is only used in bitcoin-cli, it doesn't need to be in `util.cpp` (which is shared between the server and client), would be better in a client-specific utility, e.g. `compat/noecho` or something like that, that is only linked into `-cli`.

Reply:
Good point. Fixing.